File: CPP/Sejal/SumOfLeftLeaves.cpp
// https://devsnest.in/lms?id=59&slug=sum-of-left-leaves

int solveHelp(TreeNode* root) {
    int sum = 0;
    if (root == NULL)
        return 0;
    if (root->left != NULL && root->left->left == NULL && root->left->right == NULL)
        sum += root->left->val;
    return sum + solveHelp(root->left) + solveHelp(root->right);
}

int solve(TreeNode* root){
//CODE HERE 
    int sum;
    sum = solveHelp(root);
    return sum;
}

File: java/vansh_jain/trees/sumOfLeftLeaves.java
static int solve(TreeNode root){
    //CODE HERE 
    if (root == null)
        return 0;
    if(root.left != null && root.left.left==null && root.left.right==null)
        return root.left.val + solve(root.right);
    return solve(root.left)+solve(root.right);
    
}
